lambdas/issue-credential/src/issue-credential-handler.ts

```diff
@@ -5,20 +5,104 @@ import {
   APIGatewayProxyResult,
   Handler,
 } from "aws-lambda";
-
+import { SSMClient } from "@aws-sdk/client-ssm";
+import { injectLambdaContext } from "@aws-lambda-powertools/logger";
+import errorMiddleware from "../../middlewares/error/error-middleware";
+import { AwsClientType, createClient } from "../../common/aws-client-factory";
+import { ConfigService } from "../../common/config/config-service";
+import { logger, metrics } from "../../common/utils/power-tool";
+import validateHeaderBearerTokenMiddleware from "../../middlewares/validate-header-bearer-token-middleware";
+import getSessionByAccessTokenMiddleware from "../../middlewares/session/get-session-by-access-token-middleware";
+import { DynamoDBDocument } from "@aws-sdk/lib-dynamodb";
+import getToyByIdMiddleware from "../../middlewares/toy/get-toy-by-session-id-middleware";
+import setGovUkSigningJourneyIdMiddleware from "../../middlewares/session/set-gov-uk-signing-journey-id-middleware";
+import { VerifiableCredentialBuilder } from "../../verifiable-credential/verifiable-credential-builder";
+import { VC_CONTEXT } from "../../types/verifiable-credentials";
+import { CommonConfigKey } from "../../types/config-keys";
+import initialiseConfigMiddleware from "../../middlewares/config/initialise-config-middleware";
+import { Subject } from "../../types/subject";
+import { ToyItem } from "../../types/toy_item";
+const TOY_CREDENTIAL_ISSUER = "toy_credential_issuer";
+const dynamoDbClient = createClient(AwsClientType.DYNAMO) as DynamoDBDocument;
+const ssmClient = createClient(AwsClientType.SSM) as SSMClient;
+const configService = new ConfigService(ssmClient);
+const builder = new VerifiableCredentialBuilder(ssmClient);
 export class IssueCredentialLambda implements LambdaInterface {
   public async handler(
-    _event: APIGatewayProxyEvent,
+    event: APIGatewayProxyEvent,
     _context: unknown
   ): Promise<APIGatewayProxyResult | { statusCode: number }> {
+    logger.info("Toy issue-credential lambda triggered");
+    const parameter = await configService.getParameter(
+      `/${process.env.AWS_STACK_NAME}/JwtTtlUnit`
+    );
+    //TODO: we also need the MaxJwtTtl implementation here not quite right, revisit the java version
```

Comment:
Instead of a TODO, can we remove this comment and raise it as a Jira ticket please?

Reply:
As discussed on Slack, this will be done in another PR